errors/apiErrors.ts

```diff
@@ -29,45 +56,53 @@ export function isSpecifiedError(
     code?: string;
   }
 ): boolean {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const error = (err && (err.cause as AxiosError<any>)) || err;
-  const statusCodeErr = !statusCode || error.response?.status === statusCode;
-  const categoryErr = !category || error.response?.data?.category === category;
-  const subCategoryErr =
-    !subCategory || error.response?.data?.subCategory === subCategory;
-  const errorTypeErr =
-    !errorType || error.response?.data?.errorType === errorType;
-  const codeError = !code || error.code === code;
+  if (!isHubSpotHttpError(err) && !isAxiosError(err.cause)) {
```

Comment:
Is our call to `isAxiosError(err.cause)` for backwards compatibility?

Reply:
Yeah, it's for backwards compatibility.  I flagged this as ready to review and then started changing more things so it's kind of in between state so I'm going to flip it back to draft and make the changes to the CLI simultaneously since that is the main consumer of the this package.  If I do that we can ditch a lot of this logic that was added for backwards compatability.

Reply:
Removed